src/domain/functions/answerCorrectness/answersCorrectness.js

```diff
@@ -0,0 +1,9 @@
+function answersCorrectness(question,answer){
+    if(question.rightAnswer === answer){
+        return 1;   //Rigth answer
+}        
+    return 0; //Bad answer
```

Comment:
Nie musimy sobie komplikować życia zwracaniem 0 lub 1. 
```suggestion
export function isCorrectAnswer (correctAnswer, playerAnswer) {
 return correctAnswer === playerAnswer
}
```
Bardzo szybka i miła w obsłudze funkcja :)